viewer/packages/tools/src/Geomap/Geomap.ts

```diff
@@ -28,12 +29,15 @@ export class Geomap {
   }
 
   private requestRedraw(timeOut: number) {
-    var intervalId = setInterval(() => {
-      this._viewer.requestRedraw();
-    }, 100);
+    if(this._intervalId == 0) {
+      this._intervalId = setInterval(() => {
+        this._viewer.requestRedraw();
+      }, 100);
+    }
 
     setTimeout(() => {
-      clearInterval(intervalId);
+      clearInterval(this._intervalId);
+      this._intervalId = 0;
```

Comment:
If you have a long changes of camera movement, this will only cause the _first_ movement to have any effect. I'd rather call `clearInterval` every time `_intervalId !== 0` to ensure the schedule occurs for the last camera movement event, not the first.

Reply:
`clearInterval` is called when `timeOut` value is triggered and `_intervalId == 0` or `_intervalId > 0`. If we do not want to miss huge camera movements than I can reduce the `timeOut` value for camera Movement to 1000/1500 milli seconds, which would narrow down the error